src/hooks/wallets/useOnboard.ts

```diff
@@ -43,7 +42,7 @@ export const getConnectedWallet = (wallets: WalletState[]): ConnectedWallet | nu
   const primaryWallet = wallets[0]
   if (!primaryWallet) return null
 
-  const account = primaryWallet.accounts[0]
+  const account = primaryWallet.accounts[primaryWallet.accounts.length - 1]
```

Comment:
Why do we need the last account instead of the first one now?

Reply:
Good catch, it’s an unrelated leftover from another branch. I’ll remove it.